platform.txt

```diff
@@ -107,6 +108,7 @@ build.copy_jtag_files=0
 build.openocdscript.esp32=esp32-wrover-kit-3.3v.cfg
 build.openocdscript.esp32s2=esp32s2-kaluga-1.cfg
 build.openocdscript.esp32s3=esp32s3-builtin.cfg
+build.openocdscript.esp32c2=esp32c2-builtin.cfg
```

Comment:
There is no such config file in OpenOCD, because C2 doesn't have a USB_SERIAL_JTAG peripheral.